esx_service/cli/vmdkops_admin_test.py

```diff
@@ -379,6 +410,371 @@ class TestStatus(unittest.TestCase):
     def test_status(self):
         self.assertEqual(vmdkops_admin.status(None), None)
 
+class TestTenant(unittest.TestCase):
+    """ Test tenant functionality """
+    # tenant1 info
+    tenant1_name = "test_tenant1"
+    vm1_name = 'test_vm1'
```

Comment:
@shuklanirdesh82 Could you point me what is the change you made to randomize the vm name? I want to take a look. Thanks!